### PR TITLE
Finished Seller! We can add/delete/update products

### PR DESCRIPTION
**Pull request by danielatrejo**

### CODE REVIEW

**Comment by danielatrejo** [comment]
Just merged conflicts. Now making sure everything works before merging w master

File: miniamazon-frontend/src/components/Account.js [hunk 1]
```diff
@@ -2,17 +2,10 @@ import React from 'react';
 import '../App.css';
 import axios from 'axios';
 
-var userID = "compsci316project";
```
Comment (2018ayee) [inline miniamazon-frontend/src/components/Account.js:5]: I'm getting `___ is not defined` errors for all of these variables since they are used in 44-51

Comment (danielatrejo) [reply miniamazon-frontend/src/components/Account.js:5]: me too. I am looking into it

Comment (2018ayee) [reply miniamazon-frontend/src/components/Account.js:5]: I think you have to retrieve the current user, but honestly that can be done after Martin finishes log in/sign up so you can just add these lines back

**Comment by danielatrejo** [comment]
okay I think we are Gucci now. Merge me!

File: miniamazon-frontend/src/components/Account.js (hunk 1 above)
Comment (danielatrejo) [reply miniamazon-frontend/src/components/Account.js]: fixed! and I think ready w/ last commit

**Pull request merged**